modules/dynamic-provisioning-ocs-object-definition.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following definitions:
+//
+// * storage/dynamic-provisioning.adoc
+
+[id="ocs-object-definition_{context}"]
+= Red Hat OpenShift Container Storage object definition
+
+When using Red Hat OpenShift Container Storage, the dynamic Storage classes are
```

Comment:
s/dynamic Storage classes/dynamic storage classes/

Just one more little thing^^ thanks!

Reply:
I never heard `dynamic storage class`, how about `storage classes for dynamic volume provisioning`?

Reply:
made the update per @qinpingli suggestion